libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -967,8 +967,8 @@ namespace SohImGui {
                         Tooltip("Wearing the Bunny Hood grants a speed\nincrease like in Majora's Mask");
                         EnhancementCheckbox("Fast Chests", "gFastChests");
                         Tooltip("Kick open every chest");
-                        EnhancementCheckbox("Fast Drops", "gFastDrops");
-                        Tooltip("Skip first-time pickup messages for consumable items");
+                        EnhancementCheckbox("Fast Pickups", "gFastDrops");
```

Comment:
Might be clearer if it was named "Skip pickup messages" or "Disable pickup messages"

Reply:
Much clearer, good idea. Will update next time I'm at PC.